phpugms_greeting_2017-11/index.html

```diff
@@ -131,6 +131,13 @@ <h2>December Meeting</h2>
                     <p>Speaker Wanted!</p>
                 </section>
 
+                <section>
+                    <h1>PHPTestFest Recap</h1>
+
+                    <p>8 PRs created</p>
+                    <p>a lot of Fun</p>
```

Comment:
Can you add a screenshot of https://github.com/phpcommunity/phptestfest-php-src/pulls?utf8=%E2%9C%93&q=is%3Apr%20label%3A%22UG%3A%20PHP%20M%C3%BCnster%22%20 

and maybe some Photos from our twitter account: https://twitter.com/phpugms/media

Reply:
done

Reply:
nice, thanks.